velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -29,14 +32,14 @@ class MapTest : public SparkFunctionBaseTest {
       const std::string& expression,
       const std::vector<VectorPtr>& parameters,
       bool expectException = false,
-      const VectorPtr& expected = nullptr) {
+      const VectorPtr& expected = nullptr,
+      const std::string errorMsg = "") {
```

Comment:
No need to specify both expectException flag and errorMsg. Actually, let's split this function into 2:

```
testMap(expression, parameters, expected)

testMapFails(expression, parameters, errorMessage)
```

Reply:
Updated, thanks.